src/include/migraphx/half.hpp

```diff
@@ -80,6 +80,18 @@ struct common_type<migraphx::half, migraphx::fp8::fp8e4m3fnuz>
     using type = float;
 };
 
+template <>
+struct common_type<migraphx::fp8::fp8e5m2fnuz, migraphx::half>
+{
+    using type = float;
+};
+
+template <>
+struct common_type<migraphx::half, migraphx::fp8::fp8e5m2fnuz>
+{
+    using type = float;
+};
```

Comment:
So #3606 refactors this to use the template type so these specializations are not necessary.